libwacom/libwacom.sym

```diff
@@ -61,6 +61,8 @@ global:
     libwacom_stylus_get_id;
     libwacom_stylus_get_name;
     libwacom_stylus_get_num_buttons;
+    libwacom_stylus_get_num_paired_ids;
+    libwacom_stylus_get_paired_ids;
```

Comment:
haha, found the right emoji: :rotating_light: :rotating_light: :rotating_light: 

this must go into a new section, otherwise you're changing the ABI. Basically: once a release is out, the existing sections in `libwacom.sym` must never be touched and any new additions go into the new section. See my other comment for how the section would look like, you can add those two there.

Reply:
Thanks for the tip! Wasn't quite sure how to deal with the ABI change in this new symbol-versioned reality :+1: